src/fu-util.c

```diff
@@ -2845,6 +2846,9 @@ main (int argc, char *argv[])
 	if (no_history)
 		priv->flags |= FWUPD_INSTALL_FLAG_NO_HISTORY;
 
+	/* start polkit tty agent to listen for password requests */
+	fu_polkit_agent_open ();
```

Comment:
the return code seems to be int, shouldn't GError be passed in and the return code be boolean instead (maybe with a local error)?

Reply:
Yes, good idea.

Reply:
Done,a few things cleaned up.